test/helpers/fixture.js

```diff
@@ -1,147 +1,89 @@
+const fs = require('fs');
+
 export default class Fixture {
+  // Read fixture data from the fixtures JSON file
+  static readFixtureData() {
+    const currentDir = process.cwd();
+    const fixtures = JSON.parse(fs.readFileSync(`${currentDir}/test/helpers/fixtures.json`));
+
+    return fixtures;
+  }
+
   // We keep the page_size of retrieving `all` records small so cassettes stay small
   static pageSize() {
-    return 5;
+    return this.readFixtureData().page_sizes.five_results;
   }
 
   // This is the USPS carrier account ID that comes with your EasyPost account by default and should be used for all tests
   static uspsCarrierAccountId() {
-    // Fallback to the EasyPost Node Client Library Test User USPS carrier account ID
-    return process.env.USPS_CARRIER_ACCOUNT_ID || 'ca_5ba7ca3632c54adeb17ad4bcac13c890';
+    return process.env.USPS_CARRIER_ACCOUNT_ID || '123';
   }
 
   static usps() {
-    return 'USPS';
+    return this.readFixtureData().carrier_strings.usps;
   }
 
   static uspsService() {
-    return 'First';
+    return this.readFixtureData().service_names.usps.first_service;
   }
 
   static pickupService() {
-    return 'NextDay';
+    return this.readFixtureData().service_names.usps.pickup_service;
   }
 
   static reportType() {
-    return 'shipment';
+    return this.readFixtureData().report_types.shipment;
   }
 
   // If you need to re-record cassettes, increment this date by 1
   static reportDate() {
-    return '2022-04-11';
+    return '2022-08-01';
```

Comment:
We don't want to increment the report date. That's fine being in the past.

Reply:
Weird thing to block on

Reply:
Put back four months.